test/integration.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const assert = require('assert');
+const blc = require('broken-link-checker');
+const test = require('selenium-webdriver/testing');
+const webdriver = require('selenium-webdriver');
+
+const timeOut = 35000;
```

Comment:
Is this timeout value used as a delay to wait for pages to load before executing tests? If so, I would generally try to avoid this and use some kind of CSS hook instead (in bedrock for example, we add a class of `loaded` to the `html` element, which has proven to be more robust).